setup.cfg

```diff
@@ -33,7 +33,7 @@ install_requires =
     sdcflows ~= 1.2.0
     smriprep ~= 0.5.2
     tedana >= 0.0.5
-    templateflow >= 0.4.2
+    templateflow >= 0.5.0
```

Comment:
What changes are introduced in this minor version release?

Reply:
A new template (rodent) and changes to `fsLR` / `fsaverage` templates.

Reply:
Does that affect any non-CIFTI outputs?

Reply:
I'd be surprised if they did - 99.9% sure of "no"